feat: upgrade to next 13

rin-st opened the pull request:
Everything works fine :)

technophile-04 commented:
Thanks @rin-st, tested it out and everything works like a charm!!

Just noticed a very small warning in the console(not that important though) : 
![Screenshot 2023-02-02 at 2 24 55 PM](https://user-images.githubusercontent.com/80153681/216239339-8f6ebb3a-5088-4633-9e57-e180da126561.jpg)

rin-st commented:
Thank you, @technophile-04 ! I didn't notice it. Updated pr, also changed font import. We still have a warning from `react-blockies` but they didn't approve prs for years 🤷‍♂️

carletex commented:
As discussed on DMs with @rin-st we are removing the NextJS recommended method to import a font, since we don't want to self-host it for a couple of font use in the example page.

For the main SE2 font (rn it's the default) we might want to use the Next recommended method. https://nextjs.org/docs/basic-features/font-optimization

Thanks al!!
